plugins/sync/src/main/java/info/nightscout/plugins/sync/nsclientV3/workers/LoadBgWorker.kt

```diff
@@ -64,7 +64,7 @@ class LoadBgWorker(
                         // Objective0
                         sp.putBoolean(info.nightscout.core.utils.R.string.key_objectives_bg_is_available_in_ns, true)
                         // Schedule processing of fetched data and continue of loading
-                        continueLoading = !(sgvs.size != NSClientV3Plugin.RECORDS_TO_LOAD || response.code == 304)
+                        continueLoading = !(sgvs.size <= NSClientV3Plugin.RECORDS_TO_LOAD*0.95 || response.code == 304)
```

Comment:
sorry, commited this by an accident. its a crude temporal fix for [this](https://github.com/nightscout/AndroidAPS/issues/2763)